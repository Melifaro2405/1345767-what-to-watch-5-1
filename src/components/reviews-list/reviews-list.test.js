import React from 'react';
import renderer from 'react-test-renderer';
import ReviewsList from './reviews-list';

const reviews = [
  {
    comment: `A movie that will take you to another world full of emotions.`,
    date: `2020-11-03T13:38:44.769Z`,
    id: 1,
    rating: 8.6,
    user: {
      id: 11,
      name: `Jack`
    }
  },
  {
    comment: `A movie that will take you to another world null of emotions.`,
    date: `2020-11-03T13:38:44.769Z`,
    id: 2,
    rating: 1.6,
    user: {
      id: 12,
      name: `John`
    }
  }
];

it(`Should ReviewsList render correctly`, () => {
  const tree = renderer
    .create(
        <ReviewsList
          reviews={reviews}
        />
    )
    .toJSON();

  expect(tree).toMatchSnapshot();
});
