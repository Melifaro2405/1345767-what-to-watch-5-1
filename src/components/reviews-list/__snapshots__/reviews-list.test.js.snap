// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Should ReviewsList render correctly 1`] = `
Array [
  <div
    className="review"
  >
    <blockquote
      className="review__quote"
    >
      <p
        className="review__text"
      >
        A movie that will take you to another world full of emotions.
      </p>
      <footer
        className="review__details"
      >
        <cite
          className="review__author"
        >
          Jack
        </cite>
        <time
          className="review__date"
          dateTime="2020-11-03"
        >
          November 03, 2020
        </time>
      </footer>
    </blockquote>
    <div
      className="review__rating"
    >
      8.6
    </div>
  </div>,
  <div
    className="review"
  >
    <blockquote
      className="review__quote"
    >
      <p
        className="review__text"
      >
        A movie that will take you to another world null of emotions.
      </p>
      <footer
        className="review__details"
      >
        <cite
          className="review__author"
        >
          John
        </cite>
        <time
          className="review__date"
          dateTime="2020-11-03"
        >
          November 03, 2020
        </time>
      </footer>
    </blockquote>
    <div
      className="review__rating"
    >
      1.6
    </div>
  </div>,
]
`;
